apps/src/music/blockly/FieldSounds.js

```diff
@@ -198,18 +198,20 @@ class FieldSounds extends GoogleBlockly.Field {
     // Add an image for the sound type.
     const soundType = this.options
       .getLibrary()
-      .getSoundForId(this.getValue()).type;
-
-    GoogleBlockly.utils.dom.createSvgElement(
-      'image',
-      {
-        x: 6,
-        y: 3,
-        width: 15,
-        href: `/blockly/media/music/icon-${soundType}.png`,
-      },
-      this.backgroundElement
-    );
+      .getSoundForId(this.getValue())?.type;
+
+    if (soundType) {
```

Comment:
Do we want to log when this happens, in case this is also somehow happening in the wild (or to make sure we're not affecting projects when we update libraries)?

Reply:
I added the code for it, and then observed that we have an existing warning that does the job:
https://github.com/code-dot-org/code-dot-org/blob/e4f50e71cecc856f1f43e8f1851e6830b87af88f/apps/src/music/player/sequencer/Simple2Sequencer.ts#L204-L207